libs/comparison/src/lib/difference/difference.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, OnInit } from '@angular/core';
+//import {ChangeInfo} from "@groupdocs.examples.angular/comparison";
+
+@Component({
+  selector: 'gd-comparison-difference',
+  templateUrl: './difference.component.html',
+  styleUrls: ['./difference.component.less']
+})
+export class DifferenceComponent implements OnInit {
+  // TODO: this leads to `circular dependency detected` warning
```

Comment:
to avoid `circular dependency detected` try to use  - import {ChangeInfo} from "../models";

Reply:
Yep, it works, thank you!